R/pkg/R/generics.R

```diff
@@ -387,6 +387,16 @@ setGeneric("value", function(bcast) { standardGeneric("value") })
 #' @export
 setGeneric("agg", function (x, ...) { standardGeneric("agg") })
 
+#' alias
+#'
+#' Set a new name for a Column or a SparkDataFrame. Equivalent to SQL "AS" keyword.
```

Comment:
I guess we don't say `return a new Column` but more generally `return a Column`
and in other cases we say `return a new SparkDataFrame`

so I guess it's a difference in wording.
I think what you propose is fine, though do you think it's confusing to say `Equivalent to SQL "AS" keyword.` because that makes sense only for Column and not the whole dataframe?